### DISCUSSION TITLE
[QUESTION] Cómo se relacionan las tareas hijas con la tarea padre para cambiar de estado?

### DISCUSSION THREAD

**Discussion by AlejandroLoraG** [Cotalker/documentation#292] [2. Questions and Answers] [answered]
_**Hola!**_

Cuando intento mover una tarea padre con sub tareas no pertenecientes al siguiente estado del workflow padre, el sistema me lo permita cuando no debería.

En detalle:
Cuando paso una task padre con hijas desde mi segundo estado de la statemachine, hacia el tercer estado el sistema lo permite, cuando este no debería ya que mi statemachine hija no pertenece al tercer estado de mi statemachine padre.

Statemachines:
![image](https://user-images.githubusercontent.com/51095635/123158456-cf3c6c00-d439-11eb-8d3b-9b44762e6870.png)

_**Muchas gracias!**_

**Comment by acedward** [comment, marked as answer]
@AlejandroLoraG la definición de estado hijo es sólo para la creación de nuevas subtareas. 
Una vez que una tarea tiene hijas, este de todas formas puede cambiar de estado - independiente si el estado permite (crear) subtareas.

**Reply by acedward** [reply]
Si necesitas bloquear el avance, es posible configurarlo en la rutina para cambiar de estado. E.g., revisar que todas las tareas hijas estén terminadas - ó cada vez que se termina un estado hijo, ahi revisar y avanzar al padre
